spec/javascripts/Dashboard/components/NewAccountsWidget.spec.jsx

```diff
@@ -0,0 +1,80 @@
+// @flow
+
+import React from 'react'
+
+import { NewAccountsWidget } from 'Dashboard/components/NewAccountsWidget'
+import { mount } from 'enzyme'
+
+let props
+const defaultProps = {
+  chartData: {
+    values: {
+      '2022-01-01': { value: 0, formatted_value: '0' },
+      '2022-01-02': { value: 1, formatted_value: '1' },
+      '2022-01-03': { value: 1, formatted_value: '1' }
+    },
+    complete: {
+      '2022-01-01': { value: 0, formatted_value: '0' },
+      '2022-01-02': { value: 1, formatted_value: '1' },
+      '2022-01-03': { value: 1, formatted_value: '1' }
+    },
+    incomplete: {
+      '2022-01-01': { value: 0, formatted_value: '0' },
+      '2022-01-02': { value: 1, formatted_value: '1' },
+      '2022-01-03': { value: 1, formatted_value: '1' }
+    },
+    previous: {
+      '2022-01-01': { value: 0, formatted_value: '0' },
+      '2022-01-02': { value: 1, formatted_value: '1' },
+      '2022-01-03': { value: 1, formatted_value: '1' }
+    }
+  },
+  newAccountsTotal: 10,
+  hasHistory: true,
+  links: {
+    previousRangeAdminBuyersAccount: {
+      url: '/buyers/accounts?search%5Bcreated_within%5D%5B%5D=2021-11-03&search%5Bcreated_within%5D%5B%5D=2021-12-03',
+      value: '0'
+    },
+    currentRangeAdminBuyersAccount: {
+      url: '/buyers/accounts?search%5Bcreated_within%5D%5B%5D=2021-12-03&search%5Bcreated_within%5D%5B%5D=2022-01-03'
+    },
+    lastDayInRangeAdminBuyersAccount: {
+      url: '/buyers/accounts?search%5Bcreated_within%5D%5B%5D=2022-01-03&search%5Bcreated_within%5D%5B%5D=2022-01-03',
+      value: '0'
+    }
+  },
+  percentualChange: 50
+}
+
+const mountWrapper = (props) => mount(<NewAccountsWidget {...{ ...defaultProps, ...props }}/>)
+
+it('should render', () => {
+  const wrapper = mountWrapper(defaultProps)
+  expect(wrapper.exists()).toBe(true)
+})
+
+describe('when there is historical data', () => {
+  const wrapper = mountWrapper(defaultProps)
```

Comment:
I'd include this line (and applies to every case where the `wrapper` is built this way) inside every `test` rather than inside the `describe` or I'd use a `beforeEach` to create the `wrapper` and an `afterEach` to clean it since we can't be sure if these tests will grow in complexity some time in the future.

Reply:
I ended up adding it for each test in 4b1b900.